packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/execute_event/v0/mod.rs

```diff
@@ -73,6 +78,13 @@ where
                         )
                         .map_err(Error::Drive)?;
 
+                    for validation_operation in validation_operations {
+                        let cost = validation_operation.processing_cost(platform_version)?;
+                        individual_fee_result
+                            .checked_add_assign(FeeResult::default_with_fees(0, cost))
```

Comment:
Not ideal to do it this way as checked_add_assign is more much expensive than adding, imo it would be best to add all costs for processing, then only do 1 checked_add_assign.